src/ResourceManager/Sql/Commands.Sql/ServiceObjective/Cmdlet/AzureSqlServerServiceObjectiveCmdletBase.cs

```diff
@@ -34,21 +34,7 @@ public abstract class AzureSqlServerServiceObjectiveCmdletBase
             HelpMessage = "SQL Database server name.")]
         [ValidateNotNullOrEmpty]
         public string ServerName { get; set; }
-
-        /// <summary>
-        /// Gets or sets the name of the database to use.
-        /// </summary>
-        /// <remarks>
-        /// This parameter is not needed or used. It will be removed in a future release.
-        /// </remarks>
-        [Parameter(Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            Position = 2,
-            HelpMessage = "SQL Database name.",
-            DontShow = true)]
-        [ValidateNotNullOrEmpty]
-        public string DatabaseName { get; set; }
```

Comment:
I think we need to mark `[Obsolete]` and wait one or more releases before it can be removed.

Reply:
I added the Obsolete attribute. Previously I had just set the "DontShow = true".